src/css/video-js.scss

```diff
@@ -1,4 +1,5 @@
 @import "variables";
+@import "private-variables";
```

Comment:
That'd be fine with me. Is that better for some reason?

On Mon, Sep 14, 2015 at 8:41 PM, Gary Katsevman <notifications@github.com>
wrote:

> In src/css/video-js.scss
> <https://github.com/videojs/video.js/pull/2584#discussion_r39471902>:
>
> > @@ -1,4 +1,5 @@
> >  @import "variables";
> > +@import "private-variables";
>
> what about just importing it in variables.scss?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/videojs/video.js/pull/2584/files#r39471902>.
>